### PR TITLE
Networkcloud 20240701preview version updated

### PR DESCRIPTION
**Pull request by priyamshet**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

Coresponding aaz PR https://github.com/Azure/aaz/pull/628
### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/networkcloud/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+2.0.0b7
+++++++++
+* This version requires a minimum of 2.66 Azure core CLI. See release notes for more details: https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/docs-ref-conceptual/release-notes-azure-cli.md
+* This version upgrades the internal generation tool aaz-dev-tools to 3.1.0. Refer to the release notes for more details: https://github.com/Azure/aaz-dev-tools/releases/tag/v3.1.0.
+* Optional Cluster properties can be now set to null during update (PATCH) operation. This includes `--cluster-service-principal`, `--command-output-settings`, `--compute-deployment-threshold`, `--update-strategy`, `--secret-archive`, and `--runtime-protection`. In that case, the value will be reset to the default if defined by the API.
+* Validation is added to `--command-output-settings` property of a cluster for both create and update commands. When the identityType is SystemAssignedIdentity, the UAI should not be provided.
```
Comment (mbashtovaya) [inline src/networkcloud/HISTORY.rst:11]: remove - duplicate info

**Review by mbashtovaya — CHANGES_REQUESTED** [review]
release history needs to be updated

**Comment by yonzhan** [comment]
Networkcloud

**Review by mbashtovaya — APPROVED** [review]

**Review by mbashtovaya — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Pull request merged**